hydromt_wflow/workflows/states.py

```diff
@@ -62,10 +62,11 @@ def prepare_cold_states(
         Dataset containing the staticmaps grid and variables to prepare some of the
         states.
 
-        * Required variables: wflow_subcatch, wflow_river
+        * Required variables: subcatchment, river_mask
```

Comment:
Maybe use function arguments to be fully precise?
```suggestion
        * Required variables: `mask_name_land`, `mask_name_river`
```